Cargo.toml

```diff
@@ -16,10 +16,10 @@ readme        = "README.md"
 keywords      = ["buffers", "zero-copy", "io"]
 exclude       = [
     ".gitignore",
-    ".travis.yml",
-    "deploy.sh",
+    "azure-pipelines.yml",
```

Comment:
Unrelated, but I wonder if the `exclude` is still relevant with modern Cargo. At the very least we probably should include tests?

Reply:
IIRC, (at least in modern cargo) the purpose of the `exclude` field is to prevent the size of the crate from being bloated by large files like images. So I think we can remove this. (By the way, By the way, as we do not currently have a `test` directory or `bench` directory, the current `exclude` list does little work. We have `tests` directory and `benches` directory.)

Reply:
Yeah, the exclude statement is really really old...